_posts/2019/2019-08-20-end-to-end-tests-with-geb-and-spock.md

```diff
@@ -0,0 +1,154 @@
+---
+title: End-to-end tests with Geb and Spock
+layout: post
+date: '2019-08-20'
+header_image: public/.jpg
+header_position: bottom
+header_overlay: true
+category: tech-stories
+tags: ["tech", "methods-and-tools", "coding"]
+authors: ["Thomas", "Torben"]
+about_authors: ["thirsch", "tulber"]
+---
+
+This blog post is going to be a summary of our recent endeavours to establish a solid end-to-end testing culture at ePages,
+specifically in the context of third-party service providers, e.g. payments and shippings.
+
+I will write about the motivation to start a test project, the goals that we set out to achieve, and some common problems
+and their solutions.
+
+
+## Motivation & Goals
+
+Since several years Selenium has been used at ePages. A framework was developed, with all the good intentions of categorising,
+structuring and maintaining the tests. But in the context of our third-party integrations this has been notoriously difficult.
```

Comment:
... structuring, and maintaining... (add comma)

Reply:
https://language-boutique.de/naithani/blogpost/das-oxford-komma.html
--> ist optional